web/src/main/java/com/marklogic/hub/web/service/FlowManagerService.java

```diff
@@ -287,7 +303,7 @@ private String getStepKeyInStepMap(Flow flow, String stepId) {
             String[] key = new String[1];
 
             flowManager.getSteps(flow).forEach((k, v) -> {
-                if (name.equals(v.getName()) && type.equals(v.getType().toString())) {
+                if (name.equals(v.getName()) && type.toLowerCase().equals(v.getType().toString().toLowerCase())) {
```

Comment:
Can we use this? Makes code more readable.
```suggestion
                if (name.equals(v.getName()) && type.equalsIgnoreCase(v.getType().toString())) {
```

Reply:
Yeah, it's a minor nitpick, but it makes it easier